commit 51542a6a4ce6f394d45551af6a7db9be4e79540b
Author: pigeon <fnoegip@gmail.com>
Date:   Fri Feb 2 20:51:09 2024 +0100

    Lowercase `CI_REGISTRY_IMAGE` (#1085)

    Container image names have to be lowercased to be accepted by a registry. GitLab does this for `CI_REGISTRY_IMAGE` by default.

diff --git a/src/job.ts b/src/job.ts
index b207d9a9d05d0142..1a9c345e61cbd0b9 100644
--- a/src/job.ts
+++ b/src/job.ts
@@ -164,7 +164,7 @@ export class Job {
         predefinedVariables["CI_NODE_INDEX"] = `${opt.nodeIndex}`;
         predefinedVariables["CI_NODE_TOTAL"] = `${opt.nodesTotal}`;
         predefinedVariables["CI_REGISTRY"] = `local-registry.${this.gitData.remote.host}`;
-        predefinedVariables["CI_REGISTRY_IMAGE"] = `$CI_REGISTRY/${this._variables["CI_PROJECT_PATH"]}`;
+        predefinedVariables["CI_REGISTRY_IMAGE"] = `$CI_REGISTRY/${this._variables["CI_PROJECT_PATH"]}`.toLowerCase();

         // Find environment matched variables
         if (this.environment) {
